tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py

```diff
@@ -88,7 +103,7 @@ def generateFactoryImage(self):
         # build and flash the factory image.
         if self._buildConfig.get('use_reference_bootloader', False):
             factoryImageGenCommand = \
-                'python ' + os.path.join(self._projectRootDir, '..', OtaAfrProject.OTA_FACTORY_IMAGE_GENERATOR_PATH) + ' ' + \
+                'python ' + os.path.join(self._projectRootDir, OtaAfrProject.OTA_FACTORY_IMAGE_GENERATOR_PATH) + ' ' + \
```

Comment:
Should we externalize the **python** command name into the config file? For example, the name of Python 3.0 command in my dev host is python3.

Reply:
Will look into it and try to fix it in next revision.